rethinkdb/assets/dashboards/overview.json

```
@@ -0,0 +1,534 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "RethinkDB - Overview",
```

```suggestion
    "board_title": "RethinkDB Overview",
```

Nit: Could we drop the `-` from the title? I've noticed that the url is generated from the title and will look like `/rethingdb---overview`.